app/javascript/components/patient/follow_up_flag/FollowUpFlag.js

```diff
@@ -0,0 +1,278 @@
+      this.setState({ clear_flag: true });
+    } else {
+      this.setState({ [event.target.id]: value });
+    }
+  };
+
+  handleApplyHouseholdChange = apply_to_household => {
+    const no_members_selected = apply_to_household && this.state.apply_to_household_ids.length === 0;
+    this.setState({ apply_to_household, no_members_selected });
+  };
+
+  handleApplyHouseholdIdsChange = apply_to_household_ids => {
+    const no_members_selected = this.state.apply_to_household && apply_to_household_ids.length === 0;
+    this.setState({ apply_to_household_ids, no_members_selected });
+  };
+
+  /**
+   * Handles a key press event on the search form control.
+   * Checks for enter button press and prevents submisson event.
+   * @param {Object} event
+   */
+  handleKeyPress = event => {
+    if (event.which === 13) {
+      event.preventDefault();
+    }
+  };
+
+  // Makes a POST to update the follow-up flag for the current patient.
+  submit = () => {
+    this.setState({ loading: true }, () => {
+      axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
+
+      // Different POST requests are used depending on whether this action is triggered via the dashboard or monitoree page
+      if (this.props.bulk_action) {
+        let idArray = this.props.patients.map(x => x['id']);
+        axios
+          .post(window.BASE_PATH + '/patients/bulk_edit', {
+            ids: idArray,
+            bulk_edit_type: 'follow-up',
+            follow_up_reason: this.state.follow_up_reason,
+            follow_up_note: this.state.follow_up_note,
+            clear_flag: this.state.clear_flag,
+            clear_flag_reason: this.state.clear_flag_reason,
+            apply_to_household: this.state.bulk_action_apply_to_household,
+          })
+          .then(() => {
+            location.href = window.BASE_PATH;
+          })
+          .catch(error => {
+            reportError(error);
+            this.setState({ loading: false });
+          });
+      } else {
+        axios
+          .post(window.BASE_PATH + '/patients/' + this.props.patient.id + '/follow_up_flag', {
+            follow_up_reason: this.state.follow_up_reason,
+            follow_up_note: this.state.follow_up_note,
+            clear_flag: this.state.clear_flag,
+            clear_flag_reason: this.state.clear_flag_reason,
+            apply_to_household: this.state.apply_to_household,
+            apply_to_household_ids: this.state.apply_to_household_ids,
+          })
+          .then(() => {
+            // Reload the page to see the flag on the monitoree's page
+            location.reload();
+          })
+          .catch(err => {
+            reportError(err?.response?.data?.error ? err.response.data.error : err, false);
+          });
+      }
+    });
+  };
+
+  render() {
+    return (
+      <React.Fragment>
+        <Modal.Body className="modal-follow-up-flag-body">
+          <Form.Group className="flag-radio-buttons">
```

Comment:
replace this class with `style={{ display: 'inline-flex' }}` and you should be good to go on styling

Reply:
then of course remove the `follow_up_flag.scss` file